frontend/src/components/BookingCard.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+import messageIcon from '../assets/messageIcon.svg'
+import adminIcon from '../assets/adminIcon.svg'
+
+import { useParams } from 'react-router-dom'
+import { useDispatch, useSelector } from 'react-redux'
+import { useHistory } from 'react-router-dom'
+import { RootState } from '../store/types'
+import { PATHS } from '../routes/Routes'
+import { DOMAIN_URL, ENDPOINTS } from '../store/endpoints'
+import { months, days } from '../common/dates'
+
+const BookingGroup = styled.div``
+const BookingContainer = styled.div`
+  position: relative;
+  cursor: pointer;
+  background-color: #ffffff;
+  margin: 23px;
+  min-height: 70px;
+  border-radius: 20px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  display: flex;
+  padding: 15px;
+`
+const BookingLeftDisplay = styled.div`
+  align-self: center;
+  width: 80%;
+`
+
+const DateLabel = styled.div`
+  margin-bottom: 0em;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 14px;
+  overflow: hidden;
+  color: #ff705c;
+`
+
+const TimeLabel = styled.div`
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 14px;
+`
+const BookingCCAName = styled.div`
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 14px;
+`
+
+const BookingRightDisplay = styled.div`
+  position: absolute;
+  right: 15px;
+  top: 50%;
+  transform: translate(-17%, -50%);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+const Icon = styled.img`
+  padding: 20px;
+`
+export default function BookingCard() {
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const params = useParams<{ facilityID: string }>()
+  const {
+    ViewStartDate,
+    ViewEndDate,
+    createSuccess,
+    createFailure,
+    isLoading,
+    facilityBookings,
+    selectedFacilityName,
+    selectedFacilityId,
+  } = useSelector((state: RootState) => state.facilityBooking)
+  const getHumanReadableTime = (eventStartTime: number) => {
```

Comment:
what is the output format? could it use the 2 functions in `/src/common`? (unixTo12HourTime or unixToFormattedTime) 

*if possible, help update `.substr(` to `.substring(` in both files and comment for unixToFormattedTime like how it was done in unixTo12HourTime

Reply:
Resolved in commit 9c59d802492bb5f95b587c34ae11e07f5965c73a. Output format from unix methods are different from the card.